src/radix_connect/wallet_interaction/dapp_wallet_interaction/dapp_to_wallet/interaction_unvalidated.rs

```diff
@@ -40,6 +40,18 @@ impl HasSampleValues for DappToWalletInteractionUnvalidated {
     }
 }
 
+impl DappToWalletInteractionUnvalidated {
+    pub fn sample_with_interaction_id(
```

Comment:
The `sample` and `sample_other` of `DappToWalletInteractionUnvalidated`  can now use this function `sample_with_interaction_id`.